Packs/IPQualityScore/TestPlayBooks/TestIPQualityScorePlaybook.yml

```diff
@@ -0,0 +1,304 @@
+id: 261c457c-b92a-4ad6-8927-2aaa53c28ad3
```

Comment:
please run the following demisto-sdk command to fix format issues:
```
demisto-sdk format -i Packs/IPQualityScore/TestPlayBooks/TestIPQualityScorePlaybook.yml
```
also, do we have an API key that can be used for this test playbook?